### PR TITLE
[AEROGEAR-2636] Change bind params to comply with standard bind params schema

### PR DESCRIPTION
**Pull request by JameelB**
## Description
The bind parameter for client id needs to be consistent across all APBs. We need to change the bind parameter in the `apb.yml` to comply with this standard bind params schema.

## Progress
- [x] Change the bind params to comply with the standard bind params schema for client id.

## Screenshots
image

## Verification Steps
- Successfully provisioned CRC from this branch
- Secret and service binding created as expected after binding
- Secret and service binding cleaned up successfully after unbind

## Additional Notes
Related JIRA Ticket - https://issues.jboss.org/browse/AEROGEAR-2636

### CODE REVIEW

**Review by dimitraz — APPROVED** [review]
LGTM :+1:

File: roles/bind-crc-apb/tasks/main.yml [hunk 1]
```diff
@@ -45,5 +45,5 @@
 
 - asb_encode_binding:
     fields:
-      CLIENT_NAME: "{{ CLIENT_NAME }}"
+      CLIENT_NAME: "{{ CLIENT_ID }}"
```
Comment (sedroche) [inline roles/bind-crc-apb/tasks/main.yml]: The keys needs to be updated too

**Review by maleck13 — APPROVED** [review]
Same here do we want to do some basic verification ping @dimitraz

**Comment by dimitraz** [comment]
@maleck13 will do

**Comment by dimitraz** [comment]
verifying

**Review by dimitraz — APPROVED** [review]
Verified locally:

- [x] Successfully provisioned CRC from this branch
- [x] Secret and service binding created as expected after binding
- [x] Service shows up in client config component as expected 
- [x] Secret and service binding cleaned up successfully after unbind
- [x] Service is removed from client config component as expected

**Pull request merged**